fml/platform/android/message_loop_android.cc

```diff
@@ -63,23 +88,14 @@ void MessageLoopAndroid::Run() {
   FML_DCHECK(looper_.get() == ALooper_forThread());
 
   running_ = true;
-
-  while (running_) {
-    int result = ::ALooper_pollOnce(-1,       // infinite timeout
-                                    nullptr,  // out fd,
-                                    nullptr,  // out events,
-                                    nullptr   // out data
-    );
-    if (result == ALOOPER_POLL_TIMEOUT || result == ALOOPER_POLL_ERROR) {
-      // This handles the case where the loop is terminated using ALooper APIs.
-      running_ = false;
-    }
-  }
+  LooperPrepare();
+  LooperLoop();
 }
 
 void MessageLoopAndroid::Terminate() {
   running_ = false;
   ALooper_wake(looper_.get());
```

Comment:
Should we remove this line of code? It looks like unused now, it was used to quit the while looper before.
https://github.com/flutter/engine/blob/70a1106b509ea3f34febca59967ed7a76c05ce33/fml/platform/android/message_loop_android.cc#L67-L77

Reply:
done